package.json

```diff
@@ -5,7 +5,8 @@
   "description": "Base Resin.io Karma Test Runner configuration",
   "homepage": "https://github.com/resin-io-modules/resin-config-karma",
   "scripts": {
-    "prepublish": "npm run-script lint",
+    "prepublish": "require-npm4-to-publish",
+    "prepublishOnly": "npm run-script lint",
```

Comment:
How about going for `prepack`?
As @brownjohnf mentioned in [this thread](https://www.flowdock.com/app/rulemotion/r-beginners/messages/66691) the deploy job uses resinci/npm-x86_64-alpine-node10:master which has npm v6.1.0 :)

Reply:
That might be safe to do, but I just know that this change will fix the immediate issue and has been used in all of our other modules - I'll pull the trigger on this PR and if we do decide that's the better way to go then we should go through and update all our repos using the current version